### DISCUSSION TITLE
Please open github "discussion" feature

### DISCUSSION THREAD

**Discussion by jedie** [python-cmd2/cmd2#1084] [Q&A]
I have a Questions: How can one commandset call a command from a other commandset ?

The github discussions may be a better place for this, isn't it?

EDIT: About my Question: I found a way, e.g.:
```python
class FooBarCommandSet(CommandSet):
    def do_foo(self, statement: cmd2.Statement = None):
        # ... do something ...

        # Call a other command from a other CommandSet:
        self._cmd.do_foobar(statement=statement)
```
May fail if the first command get's `statement=None` but the second needs a Statement instance, isn't it?

EDIT2: This seems to be a better idea:
```python
class FooBarCommandSet(CommandSet):
    def do_foo(self, statement: cmd2.Statement = None):
        # ... do something ...

        # Call a other command from a other CommandSet:
        return self._cmd.onecmd_plus_hooks('foobar')
```

**Comment by tleonhardt** [comment]
@jedie I've never used GitHub Discussions before, but I just enabled it for `cmd2`.  I look forward to learning more about it!

**Comment by jedie** [comment]
You can "move" this issued into discussions, too ;)

**Reply by tleonhardt** [reply]
Thanks.  I wasn't aware of that feature.  I just converted it to a discussion.
